src/allencell_ml_segmenter/thresholding/thresholding_view.py

```diff
@@ -0,0 +1,343 @@
+            self._file_input_model
+        )
+
+        layout: QVBoxLayout = QVBoxLayout()
+        layout.setContentsMargins(0, 0, 0, 0)
+        layout.setSpacing(20)
+        self.setLayout(layout)
+        layout.setAlignment(Qt.AlignmentFlag.AlignTop)
+        self.setSizePolicy(
+            QSizePolicy.Policy.Preferred, QSizePolicy.Policy.Maximum
+        )
+
+        # title
+        self._title: QLabel = QLabel("THRESHOLD", self)
+        self._title.setObjectName("title")
+        layout.addWidget(self._title, alignment=Qt.AlignmentFlag.AlignHCenter)
+
+        # selecting input image
+        self._file_input_widget: FileInputWidget = FileInputWidget(
+            self._file_input_model,
+            self._viewer,
+            self._input_files_service,
+            include_channel_selection=False,
+        )
+        self._file_input_widget.setObjectName("fileInput")
+        layout.addWidget(self._file_input_widget)
+
+        # thresholding values
+        self._threshold_label: LabelWithHint = LabelWithHint("Threshold")
+        self._threshold_label.set_hint("Values to threshold with.")
+        self._threshold_label.setObjectName("title")
+        layout.addWidget(self._threshold_label)
+
+        threshold_group_box = QGroupBox()
+        threshold_group_layout = QVBoxLayout()
+
+        # none thresholding selection
+        none_radio_layout: QHBoxLayout = QHBoxLayout()
+        self._none_radio_button: QRadioButton = QRadioButton()
+        none_radio_layout.addWidget(self._none_radio_button)
+
+        none_radio_label: LabelWithHint = LabelWithHint("None")
+        none_radio_label.set_hint("No thresholding applied.")
+        none_radio_layout.addWidget(none_radio_label)
+        threshold_group_layout.addLayout(none_radio_layout)
+
+        # specific value thresholding selection
+        specific_value_layout = QHBoxLayout()
+
+        self._specific_value_radio_button: QRadioButton = QRadioButton()
+        specific_value_layout.addWidget(self._specific_value_radio_button)
+        specific_radio_label: LabelWithHint = LabelWithHint("Specific Value")
+        specific_radio_label.set_hint(
+            "Set thresholding value you'd like to apply."
+        )
+        specific_value_layout.addWidget(specific_radio_label)
+
+        self._threshold_value_slider: QSlider = QSlider(
+            Qt.Orientation.Horizontal
+        )
+
+        self._threshold_value_slider.setRange(
+            THRESHOLD_RANGE[0], THRESHOLD_RANGE[1]
+        )  # slider values from 0 to 100 (representing 0.0 to 1.0)
+
+        self._threshold_value_spinbox: QSpinBox = QSpinBox()
+        self._threshold_value_spinbox.setRange(
+            THRESHOLD_RANGE[0], THRESHOLD_RANGE[1]
+        )
+        self._threshold_value_spinbox.setSingleStep(1)
+
+        # set default value
+        self._threshold_value_slider.setValue(
+            self._thresholding_model.get_thresholding_value()
```

Comment:
The model will just have defaults?  Or potentially have values from previous thresholding attempts?